packages/island/src/theme-default/layout/Layout/useNav.ts

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+export function useNav() {
```

Comment:
`useNav`  and `useSidebar` should put in `logic` direcotry.

Reply:
ok